### PR TITLE
CASMPET-5263 Update keycloak password process

### PR DESCRIPTION
**Pull request by kburns-hpe**
This change clarifies how to change the keycloak password and make sure that the sealed secret and customizations.yaml are updated properly.

### CODE REVIEW

**Review by rgross727 — APPROVED** [review]

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md [hunk 1]
```diff
@@ -67,24 +70,76 @@ This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the n
...
     ```
 
-8.  Encrypt the values after changing the customizations.yaml file.
+1. Encrypt the values after changing the customizations.yaml file.
+
+    ```bash
+    ./utils/secrets-seed-customizations.sh customizations.yaml
+    ```
+
+    If the above command complaints that it cannot find certs/sealed_secrets.crt then you can run the following commands to create it
```
Comment (rgross727) [inline operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md]: complaints => complains

**Review by rgross727 — APPROVED** [review]

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
+This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the non-compute node (NCN). Replace this name with the actual NCN's DNS name while executing this procedure.
 
-### Procedure
+## Procedure
 
-1.  Log in to Keycloak with the default admin credentials.
+1. Log in to Keycloak with the default admin credentials.
 
-    Point a browser at https://auth.SYSTEM_DOMAIN_NAME/keycloak/admin, replacing SYSTEM\_DOMAIN\_NAME with the actual NCN's DNS name.
+    Point a browser at `https://auth.SYSTEM_DOMAIN_NAME/keycloak/admin`, replacing SYSTEM\_DOMAIN\_NAME with the actual NCN's DNS name.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md:11]: Should this line also have cmn. added to the url?

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md [hunk 3]
```diff
@@ -1,46 +1,50 @@
...
+    ```
 
-5.  Enter the existing password, new password and confirmation, and then click **Save**.
+1. Click the **Admin** drop-down menu in the upper-right corner of the page.
+1. Select **Manage Account**.
+1. Click the **Password** tab on the left side of the page.
+1. Enter the existing password, new password and confirmation, and then click **Save**.
+1. Log on to `ncn-w001`.
+1. Download and extract the CSM tarball if you have not already done so.
+1. Change your current directory to be where you've extracted the CSM install tarball
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md:35]: missing period at end of line

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md [hunk 4]
```diff
@@ -1,46 +1,50 @@
...
+   Note: If you do not wish to download the entire CSM tarball then you can clone the `https://github.com/Cray-HPE/csm.git` repo, change the branch to `release/1.2` and copy the `vendor/stash.us.cray.com/scm/shasta-cfg/stable/utils` directory to your desired working directory and follow the steps from there.
+1. Save a local copy of the customizations.yaml file
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md:37]: missing period at end of line

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md [hunk 5]
```diff
@@ -67,24 +71,76 @@ This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the n
...
+    ```
+
+    If the above command complains that it cannot find certs/sealed_secrets.crt then you can run the following commands to create it
+
+    ```bash
+    mkdir -p certs
+    ./utils/bin/linux/kubeseal --controller-name sealed-secrets --fetch-cert > certs/sealed_secrets.crt
+    ```
+
+1. Create a local copy of the platform.yaml file
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md:87]: missing period at end of line

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md [hunk 6]
```diff
@@ -67,24 +71,76 @@ This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the n
...
 
     Give the SealedSecret controller a few seconds to update the Secret, then run the following command to see the current value of the Secret:
 
     ```bash
-    ncn-w001# kubectl get secret -n services keycloak-master-admin-auth \
+    kubectl get secret -n services keycloak-master-admin-auth \
     --template={{.data.password}} | base64 --decode
     ```
 
+1. Save an updated copy of customizations.yaml to the site-init secret in the loftsman kubernetes namespace
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md:140]: missing period at end of line

File: operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md (hunk 2 above)
Comment (kburns-hpe) [reply operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md:11]: cmn is for 1.2. The examples for 0.9 and 1.0 shouldn't have it.

**Review by gbaker-hpe — APPROVED** [review]

**Pull request merged**